charts/fluent-bit/templates/networkpolicy.yaml

```diff
@@ -7,16 +7,30 @@ metadata:
     {{- include "fluent-bit.labels" . | nindent 4 }}
 spec:
   policyTypes:
+    {{- if .Values.networkPolicy.egress.customRules }}
```

Comment:
You need to add a conditional for ingress now as it is no longer implicitly wanted if `networkPolicy.enabled` is set `true`.

You should probably also extend the primary gate to something along the lines of the below statement.

```helm
{{- if and .Values.networkPolicy.enabled .Values.networkPolicy.ingress.customRules .Values.networkPolicy.egress.customRules }}
```